EVENT.md

```diff
@@ -294,6 +296,69 @@ same-origin children, but disabled in cross-origin children.
 Navigation sources may be attributed up to 3 times. Event sources may be
 attributed up to 1 time.
 
+### Registration requests
```

Comment:
nit: this is a bit too much detail IMO. I think we can say that (paraphrasing):

1. `<a>` and `window.open` will have `navigation-source`
2. Other APIs which automatically set `Attribution-Reporting-Eligible` (like `<img>`) will contain `event-source, trigger` 
3. Request from Javascript e.g. `fetch()` can set this header manually, but it is an error for such requests to specify `navigation-source`
4. All other requests will not have the `Attribution-Reporting-Eligible` header. For those requests we still allow trigger registration.

My main concern is that we don't try to be exhaustive, since I don't want to have to update this section if we add, e.g. `<script>` support. I also don't think we need JS snippets for all those examples.